http/http/src/main/java/io/helidon/http/RequestedUriDiscoveryContext.java

```diff
@@ -154,7 +159,7 @@ public Builder trustedProxies(AllowList trustedProxies) {
          * @param discoveryTypes discovery types to use
          * @return updated builder
          */
-        @ConfiguredOption
+        @ConfiguredOption(key = "types")
```

Comment:
I think a better option would be to have a new method called `types`, and deprecate the `discoveryTypes` one. 
This way we align config key and method name, as should be the default in Helidon

Reply:
Changed.